backend/cfg/cfg_regalloc_utils.ml

```diff
@@ -332,8 +336,8 @@ let precondition : Cfg_with_layout.t -> unit =
       if num_slots <> 0
       then fatal "register class %d has %d slots(s)" reg_class num_slots)
 
-let postcondition : Cfg_with_layout.t -> unit =
- fun cfg_with_layout ->
+let postcondition : Cfg_with_layout.t -> allow_stack_operands:bool -> unit =
```

Comment:
Ah, yes, it is a vestige of the conditional nature of stack operands.
I have a feeling it might be useful in the early stage of other allocators;
I am adding a CR-soon to be sure to remove it soon-ish.